src/data_pipeline.py

```diff
@@ -176,6 +177,14 @@ def main():
     eia923_allocated = data_cleaning.identify_hourly_data_source(
         eia923_allocated, cems, year
     )
+    # Export data cleaned by above for later validation, visualization, analysis
```

Comment:
Why did you move this export up in the code? The original placement was meant to export it after the hourly data source column had been updated by the partial CEMS shaping process

Reply:
Oh never mind! The old placement was leftover from a previous iteration where the partial CEMS shaping modified this dataframe, but that's no longer the case so this move definitely makes sense!